O/openmp/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "openmp"
```

Comment:
If we go by this route, I think the name must be changed: this is _an_ implementation of OpenMP, not the only one

Reply:
what about LLVMOpenMP?

Reply:
Yes, that'd match [`LLVMLibUnwind`](https://github.com/JuliaPackaging/Yggdrasil/tree/ad0e72926f9975866440c8046e876a2e38153e38/L/LLVMLibUnwind)